examples/example-tls/pom.xml

```diff
@@ -57,6 +57,12 @@
       <version>${netty.tcnative.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <artifactId>netty-tcnative-classes</artifactId>
```

Comment:
I'd expect this not to be necessary for Maven. Remove this dependency?

Although, actually, I see you didn't update the build.gradle for this example. It is using grpc-netty-shaded. We should swap to using grpc-netty-shaded here and then we don't need to define netty-tcnative at all. I've tested that works without issue.